src/en/guide/how-to/static-redirects.md

```diff
@@ -0,0 +1,112 @@
+# "Static" Redirects
+
+> How do I configure static redirects?
+
+:::: tabs
+::: tab app.py
+```python
+### SETUP ###
+import typing
+import sanic, sanic.response
+
+# Create the Sanic app
+app = sanic.Sanic(__name__)
+
+# This dictionary represents your "static"
+# redirects. For example, these values
+# could be pulled from a configuration file.
+REDIRECTS = {
+    '/':'/hello_world',                     # Redirect '/' to '/hello_world'
+    '/hello_world':'/hello_world.html'      # Redirect '/hello_world' to 'hello_world.html'
+}
+
+# This function will return another function
+# that will return the configured value
+# regardless of the arguments passed to it.
+def get_static_function(value:object) -> typing.Callable[..., object]:
```

Comment:
`Any` is a better annotation than `object`. But since it is meant to return a route handler there is an annotation for that in `sanic.models`